main/MiNiFiMain.cpp

```diff
@@ -71,7 +71,7 @@ int main(int argc, char **argv) {
   std::string nifi_configuration_class_name = "yamlconfiguration";
   std::string content_repo_class = "filesystemrepository";
 
-  running = sem_open("MiNiFiMain", O_CREAT, 0644, 0);
+  running = sem_open("/MiNiFiMain", O_CREAT, 0644, 0);
```

Comment:
Might have to have to use a relative path here. May actually want to use MINIFI_HOME here to avoid issues.